sql/src/test/java/io/crate/integrationtests/RestSqlActionTest.java

```diff
@@ -183,10 +184,44 @@ public void testSqlRequestWithBulkArgs() throws Exception {
 
         JSONAssert.assertEquals(
                 "{\n" +
-                        "  \"cols\" : [ ],\n" +
-                        "  \"results\" : [ {\"rowcount\" : 1 }, {\"rowcount\": 1 } ],\n" +
-                        "  \"duration\" : \n" + responseDuration +
-                        "}", json, true);
+                "  \"cols\" : [ ],\n" +
+                "  \"results\" : [ {\"rowcount\" : 1 }, {\"rowcount\": 1 } ],\n" +
+                "  \"duration\" : \n" + responseDuration +
+                "}", json, true);
+    }
+
+    @Test
+    public void testSqlRequestWithCustomSchema() throws Exception {
+        restSQLExecute("{\"stmt\": \"create table foo (name string)\"}", false, "bar");
```

Comment:
Can you add an assertion to verify that the supplied schemaName is actually used and the table isn't created within the doc schema?